python_modules/libraries/dagstermill/dagstermill/solids.py

```diff
@@ -198,35 +198,11 @@ def _t_fn(step_context, inputs):
                     )
 
                 except Exception as ex:  # pylint: disable=broad-except
-                    try:
-                        with open(executed_notebook_path, "rb") as fd:
-                            executed_notebook_file_handle = (
-                                step_context.resources.file_manager.write(
-                                    fd, mode="wb", ext="ipynb"
-                                )
-                            )
-                            executed_notebook_materialization_path = (
-                                executed_notebook_file_handle.path_desc
-                            )
-                    except Exception:  # pylint: disable=broad-except
-                        step_context.log.warning(
-                            "Error when attempting to materialize executed notebook using file manager (falling back to local): {exc}".format(
-                                exc=str(serializable_error_info_from_exc_info(sys.exc_info()))
-                            )
+                    step_execution_context.log.warn(
+                        "Error when attempting to materialize executed notebook: {exc}".format(
+                            exc=str(serializable_error_info_from_exc_info(sys.exc_info()))
                         )
-                        executed_notebook_materialization_path = executed_notebook_path
-
-                    yield AssetMaterialization(
-                        asset_key=(asset_key_prefix + [f"{name}_output_notebook"]),
-                        description="Location of output notebook in file manager",
-                        metadata_entries=[
-                            EventMetadataEntry.fspath(
-                                executed_notebook_materialization_path,
-                                label="executed_notebook_path",
-                            )
-                        ],
                     )
```

Comment:
removing bc this fallback logic doesn't make sense to me. imo when the `papermill.execute_notebook` fails, we should just raise the error

Reply:
my best guess is that we were using system temp dir before which doesn't clean on process close 

but I think you are right that with tempfile.TemporaryDirectory() this doesn't make any sense